lib/standalone.js

```diff
@@ -33,6 +34,19 @@ const ALWAYS_IGNORED_GLOBS = ["**/node_modules/**", "**/bower_components/**"];
   source: string;
 }*/
 
+/*:: type resultType = {
+  config: {
```

Comment:
I think the name resultType here is confusing as there is also a stylelint$result used in the same file (and it's a completely different thing?)

If I understand the code correctly this is the result of getConfigForFile ... so is "ConfigForFileT" maybe a more appropriate name?

I'd prefer using T instead of Type (and CamelCase beginning with UpperCase) as it's consistent with CssSyntaxErrorT.